tests/core/config/test_version_migration_config.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2023 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import os
+from unittest import mock
+
+import pytest
+
+from src.taipy.core._version._version_manager import _VersionManager
+from src.taipy.core.exceptions.exceptions import NonExistingVersion
+from taipy.config.config import Config
+
+
+def test_version_migration_config_creation():
+    version_migration_config = Config.set_version_migration("version_migrations1", "latest", "latest")
```

Comment:
The usage seems too complex for what it does. I believe we can improve it.
Having to provide 3 strings (id, source, target) looks useless to me. Only one seems needed at this point.

Let's discuss it.

Reply:
For now, the default source and target version is latest.

However, when a user define a MigrationConfig, he/she should specify the source and target version so Taipy know which one to migrate.